website/docs/docs/build/documentation.md

```diff
@@ -56,33 +62,41 @@ models:
           - not_null
 
 ```
-
 </File>
 
-## Generating project documentation
 
-The default documentation experience in dbt Cloud is [dbt Explorer](/docs/collaborate/explore-projects), available on [Team or Enterprise plans](https://www.getdbt.com/pricing/). Use dbt Explorer to view your project's resources (such as models, tests, and metrics), its [metadata](/docs/collaborate/explore-projects#generate-metadata), and their lineage to gain a better understanding of its latest production state.
+### FAQs
+<FAQ path="Project/example-projects" alt_header="Are there any example dbt documentation sites?"/>
+<FAQ path="Docs/document-all-columns" />
+<FAQ path="Docs/long-descriptions" />
+<FAQ path="Docs/sharing-documentation" />
+<FAQ path="Docs/document-other-resources" />
 
-dbt Cloud developer and dbt Core users can use [dbt Docs](/docs/collaborate/build-and-view-your-docs#dbt-docs), which generates basic documentation, but doesn't offer the same speed, metadata, or visibility as dbt Explorer. 
+## Generating documentation
 
 Generate documentation for your project by following these steps:
 
-1. Run `dbt docs generate` — this command tells dbt to compile relevant information about your dbt project and warehouse into `manifest.json` and `catalog.json` files, respectively. 
-2. Ensure you've created the models with `dbt run` to view the documentation for all columns, not just those described in your project.
-3. Run `dbt docs serve` if you're developing locally to use these `.json` files to populate a local website.
+1. Run the `dbt docs generate` [command](/reference/commands/cmd-docs#dbt-docs-generate) to compile relevant information about your dbt project and warehouse into `manifest.json` and `catalog.json` files, respectively. 
+2. Ensure you've created the models with `dbt run` or `dbt build` to view the documentation for all columns, not just those described in your project.
+3. Run the `dbt docs serve` [command](/reference/commands/cmd-docs#dbt-docs-serve) if you're developing locally to use these `.json` files to populate a local website.
 
-To view a resource, its metadata, and what commands are needed in dbt Explorer, refer to [generate metadata](/docs/collaborate/explore-projects#generate-metadata) for more details.
+After adding descriptions and generating documentation, dbt provides two complementary ways to [view documentation](/docs/build/view-documentation):
 
-## FAQs
-<FAQ path="Project/example-projects" alt_header="Are there any example dbt documentation sites?"/>
-<FAQ path="Docs/document-all-columns" />
-<FAQ path="Docs/long-descriptions" />
-<FAQ path="Docs/sharing-documentation" />
-<FAQ path="Docs/document-other-resources" />
+- [**dbt Docs**:](/docs/build/view-documentation#dbt-docs) A static documentation site with model lineage, metadata, and documentation that can be hosted on S3, Netlify, or your own web server. Available on dbt Core or dbt Cloud Developer plans.
```

Comment:
Suggest moving those small bits so it doesn't read like we provide those solutions
```suggestion
- [**dbt Docs**:](/docs/build/view-documentation#dbt-docs) A static documentation site with model lineage, metadata, and documentation that can be hosted on your web server (like S3 or Netlify). Available for dbt Core or dbt Cloud Developer plans.
```